src/preparator/base_preparator.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+import pandas as pd
+
+from src.preparator import Preparator
+
+
+class BasePreparator(Preparator):
+    def __init__(self, sources: List[str], **files_kwargs) -> None:
+        super().__init__()
+        self._sources = sources
+        self._src_type = files_kwargs['src_type']
+        self._type = files_kwargs['type']
+        self._sep = files_kwargs['sep']
+        self._column_names = files_kwargs['columns']
+        self._delim_whitespace = files_kwargs['delim_whitespace']
+        self.data = pd.DataFrame()
+
+    def load_data(self) -> pd.DataFrame:
+        for src in self._sources:
+            try:
```

Comment:
Pas sûr que y ait besoin de gérer les exceptions ici...
Si y a pas les données, laisse remonter l'exception, on gérera dans service... ou fuck it on gère pas.